### ISSUE TITLE
Windows CE / Sharp Brain Program Archive

### ISSUE THREAD

**Issue by azhkprtw** [yvt/rlsf#10]
I apologize for the "Off-Topic" nature of the issue, the Windows CE Repositories were marked as "ReadOnly"

I've been working with HPC:Factor (hpcfactor.com) on archiving programs for our SCL (Basically a Windows CE Program Archive)
And with some help from another user- was able to locate some old binaries that we don't have yet archived.
Currently, we have these versions from you archived:
AgileTimbres 0.0.2
Brain Groove 2.00
Gine 0.1
Scalpel 1.31
Selector 0.32
TestCE (includes RevealT-0.1, TCExit, and TestCE 1.62)
XTBook 0.2.4 Russian
XTBook 0.2.6
BClock.zip	(EDIT, Found this One)
CeSdl.zip	(EDIT, Found this One)
CeSimplePiano.zip (EDIT, Found this One)
CeSync.zip	(EDIT, Found this one)
DrumPad.zip	(EDIT, Found this One)
Lacerator.zip	(EDIT, Found this one)
QRCodeMaker.zip	(EDIT, Found this one)
ResetKit-0.2.0.zip	(EDIT, Found this one)
WaveGen.zip	(EDIT, Found this one)

I was able to locate these files below, but unfortunately Archive.org doesn't work to show me the original descriptions of these files so we can add them to our SCL as well:
ShutUpper.zip	

Also, I found this program
https://web.archive.org/web/20121008003543/http://yvt.jp:80/works/apps/brain/BankBook but the file wasn't archived
TLDR: Basically, my question is, is there an archive website somewhere of the descriptions for these? I'm not asking you do do all of the work for me, I just wanted to see where to look so I could work on that.
